File: ClaseDI/Tema2/formulario/src/main/java/org/example/formulario/model/Usuario.java
package org.example.formulario.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Data
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Usuario {
    private String nombre,correo,apellidos,genero;
    private int edad;
    private boolean disponibilidad;

    @Override
    public String toString(){
        return nombre;
    }


}

File: Android/Tema5/TiendaNuevo/app/src/main/java/com/example/tienda2/model/Usuario.kt
package com.example.tienda2.model

import java.io.Serializable

class Usuario(var nombre: String?=null, var apellido: String?=null, var edad: Int?=0, var email:
String?=null,var password:String?=null, var id:Int?=0) : Serializable{
}